packages/amplify-util-uibuilder/src/__tests__/storageManager.test.ts

```diff
@@ -0,0 +1,43 @@
+// @aws-amplify/codegen-ui is in project dependencies
+import { StudioSchema } from '@aws-amplify/codegen-ui'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Please add `@aws-amplify/codegen-ui` as a devDependency to the `amplify-util-uibuilder`'s package.json

```suggestion
import { StudioSchema } from '@aws-amplify/codegen-ui';
```

Reply:
added as devDependency